### DISCUSSION TITLE
Centre Kanban

### DISCUSSION THREAD

**Discussion by KC-OU** [SlRvb/Obsidian--ITS-Theme#354] [Q&A]
Is there a way to centre kanbans, so, I can have 3 column and have them centred on the screen, so it can be made artistically pleasing 

![chrome_Js1aTtd93G](https://github.com/user-attachments/assets/be9d80ff-d255-4ac7-be74-7c8e2e5ab64d)

**Comment by SlRvb** [comment]
I think you want to use the Cards callout not the kanban
